File: Loiane_Training_C#/Basico/C/Ex8.cs
using System;

public class Ex8
{

    static void Main(string[] args)
    {
        Console.WriteLine("Digite o preco de 3 produtos");

        double item1 = double.Parse(Console.ReadLine());
        double item2 = double.Parse(Console.ReadLine());
        double item3 = double.Parse(Console.ReadLine());

        if (item1 < item2 && item1 < item3)
        {
            Console.WriteLine("Compre o produto 1");
        }
        else if (item2 < item1 && item2 < item3)
        {
            Console.WriteLine("Compre o produto 2");
        }
        else
        {
            Console.WriteLine("Compre o produto 3");
        }
    }
}

File: Loiane_Training_Java/Java_Basico/Exer Aula 18_19/Ex8.java
public class Ex8 {
    public static void main(String[] args) {
        int[] a = { 1, 3, 5, 7, 9, 11, 13, 15, 17, 19 };
        int[] b = { 0, 2, 4, 6, 8, 10, 12, 14, 16, 18 };
        int[] c = new int[10];

        for (int i = 0; i < a.length; i++) {
            c[i] = a[i] * b[i];
            System.out.println(c[i] + " = " + a[i] + " * " + b[i]);
        }
    }
}
